flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
```

Comment:
its used by `assert_http_server` hence names the path `is_http` to do the assertion. Other scripts return `ok` for this, and thought it was more vague. I can either make it do the same, `return "ok"` or return the path `is_http` to something like `is_http_server` or somthing. Im indefferent and was just looking to get an easy way to use `assert_http_sever` which accepts a path.